src/clients/paymentCapture/update/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+
+export declare type Capture = {
+  capture?: boolean;
```

Comment:
Como sempre vamos passar `capture: true` para essa operação, acho que removeria da tipagem para não confundir quem for usar

Reply:
exatamente, seria melhor nem existir a tipagem nesse ponto

Reply:
feito